Tests/FoundationEssentialsTests/DataTests.swift

```diff
@@ -2276,6 +2276,62 @@ extension DataTests {
     }
 
+    @Test func testBase64LineLengthOptions() {
+        let expected46 = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=="
+        let length46String = Data(repeating:0, count: 46).base64EncodedString(options: .lineLength64Characters)
+        #expect(length46String == expected46)
+        let length46Data = Data(repeating:0, count: 46).base64EncodedData(options: .lineLength64Characters)
+        #expect(length46Data.count == 64)
+        #expect(String(decoding: length46Data, as: Unicode.UTF8.self) == expected46)
+
+        let expected47 = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
+        let length47String = Data(repeating:0, count: 47).base64EncodedString(options: .lineLength64Characters)
+        #expect(length47String == expected47)
+        let length47Data = Data(repeating:0, count: 47).base64EncodedData(options: .lineLength64Characters)
+        #expect(length47Data.count == 64)
+        #expect(String(decoding: length47Data, as: Unicode.UTF8.self) == expected47)
+
+        let expected48 = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+        let length48String = Data(repeating:0, count: 48).base64EncodedString(options: .lineLength64Characters)
+        #expect(length48String == expected48)
+        let length48Data = Data(repeating:0, count: 48).base64EncodedData(options: .lineLength64Characters)
+        #expect(length48Data.count == 64)
+        #expect(String(decoding: length48Data, as: Unicode.UTF8.self) == expected48)
+
+        let length49 = Data(repeating:0, count: 49).base64EncodedString(options: .lineLength64Characters)
+        #expect(length49 == #"AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\#r\#nAA=="#)
+        #expect(Array(length49.utf8)[64] == 13)
+        #expect(Array(length49.utf8)[65] == 10)
+    }
+
+    // we have more encodeToStringTests than we have encodeToDataTests.
+    // lets fix this by ensuring data output matches string output.
+    @Test func testBase64DataOutputMatchesStingOutput() {
+        for count in 0..<10_000 {
+            let data = Data(repeating: 0, count: count)
+            let stringBase64 = data.base64EncodedString(options: .lineLength64Characters)
+            let dataBase64 = data.base64EncodedData(options: .lineLength64Characters)
+
+            #expect(stringBase64 == String(decoding: dataBase64, as: Unicode.UTF8.self))
+        }
+
+        for count in 0..<10_000 {
```

Comment:
Why 10,000? If it takes a while to run and doesn’t provide additional code coverage it might be better to pick a lower number.